docs/en/queries/transactions.md

```diff
@@ -38,12 +38,14 @@ try {
         [3, 'db'],
     ];
         
-    foreach ($tags as [$id, $name] => $tag) {
+    foreach ($tags as list($id, $name)) {
         $insertTagCommand->execute();
         $insertPostTagCommand->execute();
     }    
-} catch () {
+} catch (Exception $e) {
     $transaction->rollBack();
+    // to get a slightest info about the Exception
+    var_dump($e->getMessage());
```

Comment:
That's a no go. Echoing exception isn't a good idea.

Reply:
Yeah maybe commenting it and saying to use it only for development would be needed. It is a no go for prod.